app/install/aws-cloudformation.md

```diff
@@ -73,6 +75,36 @@ on AWS RDS for you.
 - [ap-southeast-2]({{ page.links.aws }}?region=ap-southeast-2#/stacks/new?stackName=kong-elb-postgres-pv&templateURL={{ page.links.templates.kong-postgres-pv }})
 - [sa-east-1]({{ page.links.aws }}?region=sa-east-1#/stacks/new?stackName=kong-elb-postgres-pv&templateURL={{ page.links.templates.kong-postgres-pv }})
 
+#### Kong in DB-less mode
+
+Provisions Kong resources in a new VPC or existing VPC. 
+
+Note: User would need to provide a S3 bucket location where `kong.yml` is
```

Comment:
```suggestion
Note: User would need to provide an S3 bucket location where `kong.yml` is
```

Reply:
Its not a vowel Why an?

Reply:
It's not actually vowel vs consonant but _vowel _sound__ vs _consonant _sound__ , S3 begins with a vowel sound

Reply:
We say it as "an ess-three bucket."

https://docs.aws.amazon.com/AmazonS3/latest/user-guide/create-bucket.html